bucket/diffinity.json

```
@@ -0,0 +1,27 @@
+{
+    "version": "0.9",
+    "description": "Diff and merging tool with focus on accurate and easy to read source code diffs.",
+    "homepage": "http://www.truehumandesign.se/s_diffinity.php",
+    "license": {
+        "identifier": "Freeware",
+        "url": "http://www.truehumandesign.se/licenseDiffinity.php"
+    },
+    "url": "http://www.truehumandesign.se/download/diffinityInstaller.exe#/dl.7z",
+    "hash": "c6222897140226dac1ca309577126c51c486853288e5c2f62d0edab12b0988bf",
+    "pre_install": [
+        "if (!(Test-Path \"$dir\\diffinity.cfg\")) { New-Item \"$dir\\diffinity.cfg\" -ItemType File | Out-Null }",
```

Sorry missed this one. See previous review for explanation.

```suggestion
        "if (!(Test-Path \"$persist_dir\\diffinity.cfg\")) { New-Item \"$dir\\diffinity.cfg\" -ItemType File | Out-Null }",
```